CHANGELOG_PENDING.md

```diff
@@ -1,4 +1,5 @@
 ### Improvements
+* Adds CI detector for Buildkite
```

Comment:
Please also update this to include a link to the pull request (look at CHANGELOG.md for reference)

Reply:
Done, thank you.